README.md

```diff
@@ -6,32 +6,65 @@
 This module is IaC - infrastructure as code which contains a nomad job of [hive](https://hive.apache.org/).
 
 ## Content
-1. [Usage](#usage)
+0. [Prerequisites](#prerequisites)
+1. [Compatibility](#compatibility)
 2. [Requirements](#requirements)
     1. [Required software](#required-software)
-    2. [Providers](#providers)
-3. [Inputs](#inputs)
-4. [Outputs](#outputs)
-5. [Modes](#modes)
-6. [Example usage](#example-usage)
+3. [Usage](#usage)
+   1. [Providers](#providers)
+   2. [Intentions](#intentions)
+4. [Inputs](#inputs)
+5. [Outputs](#outputs)
+6. [Modes](#modes)
+7. [Example](#example)
     1. [Verifying setup](#verifying-setup)
         1. [Data example upload](#data-example-upload)
-7. [License](#license)
+8. [Authors](#authors)
+9. [License](#license)
+10. [References](#references)
 
-## Usage
-The following command will run a instance of hive found in the [example](/example) folder.
-```sh
-make test
-```
+## Prerequisites
+Please follow [this section in original template](https://github.com/fredrikhgrelland/vagrant-hashistack-template#install-prerequisites)
+
+## Compatibility
+|Software|OSS Version|Enterprise Version|
+|:---|:---|:---|
+|Terraform|0.13.1 or newer||
+|Consul|1.8.3 or newer|1.8.3 or newer|
+|Vault|1.5.2.1 or newer|1.5.2.1 or newer|
+|Nomad|0.12.3 or newer|0.12.3 or newer|
 
 ## Requirements
+
 ### Required software
-- [GNU make](https://man7.org/linux/man-pages/man1/make.1.html)
+All software is provided and run with docker.
+See the [Makefile](Makefile) for inspiration.
+
+## Usage
+The following command will run hive in the [example/standalone](example/standalone) folder.
+```sh
+make up
+```
 
 ### Providers
 - [Nomad](https://registry.terraform.io/providers/hashicorp/nomad/latest/docs)
 - [Vault](https://registry.terraform.io/providers/hashicorp/vault/latest/docs)
 
+### Intentions
+Service mesh [communication controlled by intentions](https://learn.hashicorp.com/tutorials/consul/get-started-service-networking#control-communication-with-intentions).
```

Comment:
Can you make this a more descriptive sentence? I'm not sure what this means.

Reply:
This is straight forward IMHO.
```
Service mesh communication controlled by intentions. 
Intentions are required when consul acl is enabled and default_policy is deny. 
```

Did u mean part about `Service mesh` (WHAT is it? WHY? LINKS?) or smth else?

Reply:
More like a build-up to why I would use that link.

Reply:
Is it better described in https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/43/commits/cfe214db12c728da9c672076ed366b3cccbe9a2f ?

Reply:
Just to chime in, I like the one you linked better, Nikita. It gives some small context.